src/crimson/common/fatal_signal.cc

```diff
@@ -78,27 +88,36 @@ void FatalSignal::install_oneshot_signal_handler()
   assert(r == 0);
 }
 
-
 [[gnu::noinline]] static void print_backtrace(std::string_view cause) {
-  std::cerr << cause;
-  if (seastar::engine_is_ready()) {
-    std::cerr << " on shard " << seastar::this_shard_id();
-  }
   // nobody wants to see things like `FatalSignal::signaled()` or
   // `print_backtrace()` in our backtraces. `+ 1` is for the extra
   // frame created by kernel (signal trampoline, it will take care
   // about e.g. sigreturn(2) calling; see the man page).
   constexpr std::size_t FRAMES_TO_SKIP = 3 + 1;
-  std::cerr << ".\nBacktrace:\n";
-  std::cerr << boost::stacktrace::stacktrace(
+
+  // Let's inform regarding the abort before getting the stacktrace
+  GENERIC_ERROR("Aborting {} on shard {} - Stopping all shards",
+    cause,
+    seastar::engine_is_ready() ? std::to_string(seastar::this_shard_id()) : "no shard");
+
+  seastar::engine().exit(1);
+
```

Comment:
See above, sometimes it seems that parsing the stacktrace is extremely slow (it didn't stop in my env, yet). Can we also print raw stacktraces to `std::cerr` (and to seastar logger, perhaps) before calling `boost::stacktrace::to_string()` ?